modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+
+require_dependency 'va_facilities/application_controller'
+require_dependency 'va_facilities/pagination_headers'
+require_dependency 'va_facilities/geo_serializer'
+require_dependency 'va_facilities/csv_serializer'
+
+module VaFacilities
+  module V1
+    class NearbyController < ApplicationController
```

Comment:
Should this be its own controller, or should nearby be an action on a Facilities controller or something?

Reply:
I made this its own controller because with creating a new version and endpoint it seemed simplest to keep things separate. I know that is not a satisfying answer. :smile: If it makes more sense to rename nearby_controller to be the v1 of the facilities_controller I'm fine with it.

Reply:
personally I prefer nearby as a controller with an `index` action as opposed to an action called `nearby` on a controller

Reply:
Ok! I was just concerned about how routing would look, but it's all under the facilities api anyway